pkg/cmd/roachtest/roachstress.sh

```diff
@@ -16,6 +16,66 @@ if [ -z "${BASH_VERSINFO}" ] || [ -z "${BASH_VERSINFO[0]}" ] || [ "${BASH_VERSIN
   exit 1
 fi
 
+function help {
+  cat <<EOF
+Stress run roachtest
+Usage:
+  $(basename ${0}) [flags] [-- roachtest flags]
+
+flags:
+  -t - test to run (overrides TEST)
```

Comment:
I think this should be a positional argument rather than a flag, i.e. `roachstress.sh tpcc/headroom/n4cpu16`.